examples/systemToSystemLocalForeignAssets.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetsTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { AssetsTransferApi, constructApiPromise } from '../src';
+import { TxResult } from '../src/types';
+import { GREEN, PURPLE, RESET } from './colors';
+
+/**
+ * In this example we are creating a call to send foreign asset '{"parents":"1","interior":{"X2":[{"Parachain":"2125"},{"GeneralIndex":"0"}]}}'
+ * from a Kusama Asset Hub (System Parachain) account
+ * to a Kusama Asset Hub (System Chain) account, where the `xcmVersion` is set to 2, and the `isLimited` declaring that
```

Comment:
Should use in both cases "System Parachain" so the user does not think it is a  different chain